webview/src/experiments/components/table/Cell.tsx

```diff
@@ -189,15 +190,22 @@ export const CellWrapper: React.FC<
     cellId: string
     children?: React.ReactNode
   }
-> = ({ cell, cellId, changes }) => (
-  <div
-    {...cell.getCellProps({
-      className: cx(styles.td, cell.isPlaceholder && styles.groupPlaceholder, {
-        [styles.workspaceChange]: changes?.includes(cell.column.id)
-      })
-    })}
-    data-testid={cellId}
-  >
-    {cell.render('Cell')}
-  </div>
-)
+> = ({ cell, cellId, changes }) => {
+  return (
+    <div
+      {...cell.getCellProps({
+        className: cx(
+          styles.td,
+          cell.isPlaceholder && styles.groupPlaceholder,
```

Comment:
It doesn't really matter, but since we have a block with conditionals just below, this could be moved there as `[styles.groupPlaceholder]: cell.isPlaceholder`